.github/workflows/main.yml

```diff
@@ -25,19 +30,31 @@ jobs:
         fetch-depth: 10
 
     - name: Prepare (Linux)
-      if: matrix.os == 'ubuntu-16.04'
+      if: matrix.os == 'ubuntu-16.04' && matrix.container != 'alpine'
       run: |
         sudo apt-get install libtool-bin libtommath0 libtommath-dev libicu-dev zlib1g-dev
 
     - name: Build (Linux)
-      if: matrix.os == 'ubuntu-16.04'
+      if: matrix.os == 'ubuntu-16.04' && matrix.container != 'alpine'
       run: |
         CC=clang CXX=clang++ ./autogen.sh --enable-binreloc --with-builtin-tomcrypt --prefix=/opt/firebird
         make
         make dist
         tar xzvf gen/Firebird-[0-9]*.tar.gz
         (cd Firebird-[0-9]*; sudo ./install.sh -silent)
 
+    - name: Prepare (Linux, Alpine)
+      if: matrix.os == 'ubuntu-18.04' && matrix.container == 'alpine'
+      run: apk update && apk --no-cache --update add build-base libtool git autoconf automake zlib-dev icu-dev ncurses-dev libedit-dev linux-headers tar
+
+    - name: Build (Linux, Alpine)
+      if: matrix.os == 'ubuntu-18.04' && matrix.container == 'alpine'
+      run: |
+        ./autogen.sh --enable-binreloc-threads --with-builtin-tomcrypt -with-builtin-tommath --prefix=/opt/firebird
```

Comment:
Dont this need to be "--with-builtin-tommath" (two "-")?

Reply:
Oh, sure. But it recognize it even with one `-`. But as I can see, there is no need in tomcrypt.
```
Running ./configure --enable-binreloc-threads --with-builtin-tomcrypt -with-builtin-tommath --prefix=/opt/firebird
configure: WARNING: unrecognized options: --with-builtin-tomcrypt
```
I'll remove tomcrypt from here